server/src/internalClusterTest/java/org/opensearch/index/ShardIndexingPressureSettingsIT.java

```diff
@@ -645,6 +645,14 @@ public void testShardIndexingPressureLastSuccessfulSettingsUpdate() throws Excep
                 IndexingPressureService.class,
                 coordinatingOnlyNode
             ).getShardIndexingPressure().getShardIndexingPressureTracker(shardId);
+            assertBusy(
```

Comment:
Could you refactor this to a static helper like the following? I might make the code a bit more concise and easier to read.

```
private static void waitForTwoOutstandingRequests(ShardIndexingPressureTracker tracker) {
...
}
```

Reply:
Updated